Assets/Scripts/Models/Character.cs

```diff
@@ -376,7 +376,7 @@ public void WriteXml(XmlWriter writer)
         foreach (Need n in needs)
         {
             int storeAmount = (int)(n.Amount * 10);
-            needString = needString + n.needType + ";" + storeAmount.ToString() + ":";
+            needString = needString + n.Type + ";" + storeAmount.ToString() + ":";
```

Comment:
I was thinking: "It is a little dully to have `needType` in `Need` class."
I mean, it is quite obvious that `Type` that is property of `Need`, is - `NeedType`